src/app/aircraft/add/index.njk

```diff
@@ -79,7 +79,7 @@
              <label class="govuk-label" for="craftbase">Aircraft home port / location</label>
           {{ m.error_message('craftbase', errors) }}
           <span class="govuk-hint">
-            Provide the IATA / ICAO code or coordinates (decimal). You are only required to enter coordinates if your port does not have an IATA/ICAO code. Provide coordinates to 4 decimal places.
```

Comment:
Verified locally and all looks good James. The only comment I don't find implemented is David suggestion for secition 3 of 8, that once an airport is added to the GAR, the following sentence should really disappear as it is no longer relevant for this GAR: "Add the details of a new aircraft you have not already saved". I still see this sentence once I have added an aircraft. can you check please?